### PR TITLE
NEW: (#1) Create a basic website with information on D-Nitro

### PR DESCRIPTION
**Pull request by mattiassluis**

### CODE REVIEW

File: 404.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+title: "Darn.. something must have gone wrong"
```
Comment (mattiassluis) [inline 404.md:2]: Ouch.. this needs fixing, this was copy-pasted but not updated

File: 50x.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+title: "Blipz! Something went terribly wrong"
+published: true
+classes: wide
+---
+
```
Comment (mattiassluis) [inline 50x.md:6]: Ouch.. this needs fixing, this was copy-pasted but not updated

File: README.md [hunk 3]
```diff
@@ -25,14 +25,37 @@ by creating a pull request to this repository.
...
+
 ### Adding new projects
 New projects maintained by the D-Nitro team should be added to the website
 (and also be linked on the KPN Github page).
 
 ### Testing locally
 You can test your changes locally by running a development server::
 
-    $ jekyll serve --livereload
+    $ bundle exec jekyll serve --livereload
```
Comment (reindrich) [inline README.md:59]: The first time I needed to run `$ bundle install`, and then I could run `$ bundle exec jekyll serve --livereload`

File: _projects/katka.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+## Integrations
+
+We want to make Katka as flexible as possible when it comes to the tools that
+are being used. At first we will of course create integrations with our primary
+tools and services but it should be easy to integrate with others like GitHub,
+Gitlab, Travis, etc.
+
+## Automation
+
+Release processes can be very painful, especially in highly regulated
```
Comment (reindrich) [inline _projects/katka.md:25]: I will leave the last part of the sentince out (like our own)

**Review by reindrich — COMMENTED** [review]
Two small comments, but overall it looks good!

File: README.md (hunk 3 above)
Comment (mattiassluis) [reply README.md:59]: I have added instructions for this

File: _projects/katka.md (hunk 4 above)
Comment (mattiassluis) [reply _projects/katka.md:25]: Fixed

**Pull request merged**